app/models/twilio_redirection.rb

```diff
@@ -0,0 +1,31 @@
+class TwilioRedirection
+  def self.for(twilio_number)
+    if (location = ::Location.current.find_by(twilio_number: twilio_number))
```

Comment:
Does Rubocop complain if we don't parenthesise this expression? I think we should settle on a standard for assignments in boolean expressions. I have no problem with them, nor when they are not surrounded by parens. I appreciate that it calls out that this is an assignment but perhaps if we don't spot that, we have bigger problems 💃 

@dwhenry @benbarnett any thoughts?